src/main/include/log4cxx/log4cxx.h.in

```diff
@@ -29,10 +29,14 @@
 #define LOG4CXX_VERSION_MINOR @log4cxx_VERSION_MINOR@
 #define LOG4CXX_VERSION_PATCH @log4cxx_VERSION_PATCH@
 #define LOG4CXX_VERSION_TWEAK @log4cxx_VERSION_TWEAK@
-#define LOG4CXX_MAKE_VERSION(major, minor, patch, tweak) ((major << 24) |\
-	(minor << 16) |\
-	(patch << 8) |\
+#define LOG4CXX_MAKE_VERSION(major, minor, patch, tweak) (((major) << 24) |\
+	((minor) << 16) |\
+	((patch) << 8) |\
```

Comment:
the parentheses are not needed, since you should only pass in constants to `LOG4CXX_MAKE_VERSION` in the first place: `LOG4CXX_MAKE_VERSION(0,0,12,0)` for example.  They don't hurt at least.